scripts/lib/exec.ts

```diff
@@ -27,50 +27,29 @@ export async function exec(
 ): Promise<string> {
 	nicelog(`> $ ${command} ${args.join(' ')} (in ${pwd}))`)
 	return new Promise((resolve, reject) => {
-		const data: string[] = []
-
-		const childProcess = execFile(
-			command,
-			args.filter((arg): arg is string => !!arg),
+		const childProcess = execute(
+			`${command} ${args.filter((arg): arg is string => !!arg).join(' ')}`,
 			{ cwd: pwd, env: { ...process.env, ...env } },
-			(err) => {
-				if (err) reject(err)
-				else resolve(data.join(''))
+			(err, stdout, stderr) => {
+				if (err) {
+					reject(err)
+				} else {
+					const combinedOutput = `${stdout}${stderr}`
+					resolve(combinedOutput.trim())
```

Comment:
We want `stdout` and `stderr` to be interleaved rather than one after the other. Why this change?